strategies/beaconblockproposal/best/score.go

```diff
@@ -14,680 +14,30 @@
 package best
 
 import (
-	"bytes"
 	"context"
-	"fmt"
-	"sort"
+	"math/big"
 
 	"github.com/attestantio/go-eth2-client/api"
-	apiv1deneb "github.com/attestantio/go-eth2-client/api/v1/deneb"
-	"github.com/attestantio/go-eth2-client/spec"
-	"github.com/attestantio/go-eth2-client/spec/altair"
-	"github.com/attestantio/go-eth2-client/spec/bellatrix"
-	"github.com/attestantio/go-eth2-client/spec/capella"
-	"github.com/attestantio/go-eth2-client/spec/deneb"
-	"github.com/attestantio/go-eth2-client/spec/phase0"
-	"github.com/prysmaticlabs/go-bitfield"
 )
 
 // scoreBeaconBlockPropsal generates a score for a beacon block.
-// The score is relative to the reward expected by proposing the block.
-func (s *Service) scoreBeaconBlockProposal(ctx context.Context,
+// The score is the reward expected by proposing the block.
+func (*Service) scoreBeaconBlockProposal(_ context.Context,
```

Comment:
Unless I'm missing something I think this is only called in one place so we can probably remove the unused context argument.

Reply:
Because this is a backport of existing code we want to keep this for now.  It can be revisited in master.